### ISSUE TITLE
TSUN informs about a SyntaxError after each command

### ISSUE THREAD

**Issue by wissenstein** [HerringtonDarkholme/typescript-repl#24]
Environment: Windows 10, Node.js 7.7.3, typescript 2.2.1, tsun 0.3.7

Start tsun.
Type the following line:
var a: string = 'b';
Press Enter key.

ACTUAL RESULT
The following message appears:
```
evalmachine.<anonymous>:1
{"version":3,"file":"TSUN.repl.generated.js","sourceRoot":"","sources":["../../TSUN.repl.generated.ts"],"names":[],"mappings":"AAAA,4HAA4H;AAE5H,IAAI,CAAC,GAAW,GAAG,CAAC"}
          ^
SyntaxError: Unexpected token :
    at createScript (vm.js:53:10)
    at Object.runInContext (vm.js:86:10)
    at startEvaluate (C:\Users\Oleksii_Lupandin\AppData\Roaming\npm\node_modules\tsun\bin\src\repl.js:145:25)
    at replLoop (C:\Users\Oleksii_Lupandin\AppData\Roaming\npm\node_modules\tsun\bin\src\repl.js:169:9)
    at C:\Users\Oleksii_Lupandin\AppData\Roaming\npm\node_modules\tsun\bin\src\repl.js:307:9
    at Interface._onLine (readline.js:236:5)
    at Interface._line (readline.js:582:8)
    at Interface._ttyWrite (readline.js:861:14)
    at ReadStream.onkeypress (readline.js:117:10)
    at emitTwo (events.js:106:13)
```

EXPECTED RESULT
value of a is confirmed, no error occurs.

**Comment by HerringtonDarkholme** [comment]
I cannot reproduce it locally.

I suspect this is triggered by your project setting if you are starting tsun in TS project.

If possible, please provide the tsconfig.json .

**Comment by wissenstein** [comment]
Yes, you are right, outside of the project folder it works.
tsconfig.json content is as follows:
```
{
  "compileOnSave": false,
  "compilerOptions": {
    "outDir": "./dist/out-tsc",
    "sourceMap": true,
    "declaration": false,
    "moduleResolution": "node",
    "emitDecoratorMetadata": true,
    "experimentalDecorators": true,
    "target": "es5",
    "lib": [
      "es2016",
      "dom"
    ]
  }
}
```

**Comment by HerringtonDarkholme** [comment]
Fixed in 0.3.9
